streamlit_authenticator/authenticate.py

```diff
@@ -1,8 +1,9 @@
+from datetime import datetime, timedelta
```

Comment:
This is just for visual purposes, but please arrange import lines in increasing order of character length as follows:

import ......
import ............
import ..................
import .........................